base/pkg.jl

```diff
@@ -13,9 +13,17 @@ type PkgError <: Exception
     msg::AbstractString
 end
 
-for file in split("dir types reqs cache read query resolve write entry git")
-    include("pkg/$file.jl")
-end
+include("pkg/dir.jl" )
```

Comment:
It's mostly a style thing to me.

Nowhere else in Base does anything like this, and it's generally done with explicit `include` statements in the rest of the ecosystem. It's also weird that it's done all in one string? Do you have an argument for the way it is now?